src/js/lib/monster.apps.js

```diff
@@ -311,7 +311,7 @@ define(function() {
 					if (cancelCall) {
 						return errorCallback && errorCallback();
 					} else {
-						var apiSettings = $.extend({
+						var apiSettings = _.assignIn({
```

Comment:
I used [`lodash#assignIn`](https://lodash.com/docs/4.17.11#assignIn) because [it was not a deep copy](https://api.jquery.com/jquery.extend/), in order to not change the current behavior (just in case).

Reply:
👍 

Maybe leave a comment explaining why `lodash#assignIn` is used here since we will usually prefer using `merge`/`assign`.

Reply:
:+1: I'll add it